content/committees/specification/guide/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Guide to the Jakarta EE Specification Process"
+date: "2021-03-04T00:00:00+00:00"
+layout: "single"
+aliases:
+    - committees/specification/guide/
+---
+
+## Content
+
+- [Introduction]({{< ref "#intro" >}})
+- [Creation Review]({{< ref "#creation" >}})
+- [Plan Review]({{< ref "#plan" >}})
+- [Progress Review]({{< ref "#progress" >}}) *(optional)*
+- [Release Review]({{< ref "#release" >}})
+- [Service Release]({{< ref "#service" >}})
+
+## Introduction {#intro}
+
+This guide is a step-by-step practical guide to help Jakarta EE Specification projects navigate the steps of the Jakarta EE Specification Process ([JESP](https://jakarta.ee/about/jesp/)). 
+It is meant as a helpful tool, so **if in doubt, alway refer to the [JESP](https://jakarta.ee/about/jesp/) for completeness**. Please do not hesitate to contact the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) if you have any questions or comments about anything related to the development of Jakarta EE Specifications.
+
+## Creation Review {#creation}
+
+![Creation Review](JESP_creation-review.png)
```

Comment:
Should each of these review modules have a direct link to the specific section of the JESP/EFSP that describes the process in more detail?  I know we have a general link to refer back to the JESP.  But, I'm wondering if more direct links would be helpful.